lib/services/cacheUtils.js

```diff
@@ -49,7 +49,7 @@ function createCache() {
                 stdTTL: config.authentication.cacheTTLs.users
             }),
             validation: new NodeCache({
-                stdTTL: config.authentication.cacheTTLs.users
+                stdTTL: config.authentication.cacheTTLs.validation
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-pep-steelskin/pull/348/commits/1b393f4991f291fe47439b592ea2359127274abd